### PR TITLE
Perf: zip export

### PR DESCRIPTION
**Pull request by younesdessia**
* **What kind of changes does this PR introduce?** 
- [ ] Bug fix
- [ ] new features
- [x] performance 
- [ ] docs update

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
- [ ] Yes:
- [x] No

* **Other information**:

### CODE REVIEW

File: dessia_common/core.py [hunk 1]
```diff
@@ -557,13 +557,20 @@ def zip_settings(self):
         Excel file stream generated by calling 'to_xlsx_stream' method.
         JSON file stream generated by calling 'save_to_stream' method.
         """
-        excel_stream = dcf.BinaryFile("excel_export.xlsx")
-        self.to_xlsx_stream(excel_stream)
-
-        json_stream = dcf.JsonFile("json_export.json")
-        self.save_to_stream(json_stream)
+        list_stream = []
```
Comment (GhislainJ) [inline dessia_common/core.py:560]: Avoid stating types in variable names, consider `streams`

File: dessia_common/core.py [hunk 2]
```diff
@@ -557,13 +557,20 @@ def zip_settings(self):
...
+        for export_format in self._export_formats():
+            if export_format.extension != "zip":
+                method_name = export_format.method_name
+                if export_format.text:
```
Comment (GhislainJ) [inline dessia_common/core.py:564]: This statements can be simplified. Consider this, instead : 

```py
stream_class = dcf.StringFile if export_format.text else dcf.BinaryFile
stream = stream_class(filename=f"export.{export_format.extension}")
getattr(self, method_name)(stream)
streams.append(stream)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -18,6 +18,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Refactor
 
+- Export: Enhanced .zip export
```
Comment (GhislainJ) [inline CHANGELOG.md:21]: Please state the actual changes instead of unclear "Enhanced"

**Review by GhislainJ — CHANGES_REQUESTED** [review]

File: dessia_common/core.py (hunk 2 above)
Comment (younesdessia) [reply dessia_common/core.py]: Yes !
done !

File: CHANGELOG.md (hunk 3 above)
Comment (younesdessia) [reply CHANGELOG.md]: done !

File: dessia_common/core.py (hunk 1 above)
Comment (younesdessia) [reply dessia_common/core.py]: done !

**Review by GhislainJ — APPROVED** [review]

**Pull request merged**